### PR TITLE
Added : Wishlist page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (58)

### CODE REVIEW

File: Wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <div class="nav">
+        <div class="nav-brand ml-5">
+            <div class="nav-logo">Logo</div> 
+                <a href="#"><p class="brand">Airave Mart</p></a>
+            </div>
+            <div class="searchbox mr-2">
+                <input type="text" class="search-input">
+                 <i class="bi bi-search search-icon"></i>
+             </div>
+        <ul class="mr-5 gap-1">
```
Comment (pankaj-prog) [inline Wishlist/wishlist.html]: use semantic tags

File: Wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <img src="/assest/adidas.jpg" class="card-img" alt="card-image"/>
+            <div class="card-body">
+                <div class="card-title">American Tourister</div>
+                <div class="card-description">This is a new quality bag.</div>
+                <div class="price">Rs. 1234</div>
+                <a href="/Cart/cart.html"><button class="btn btn-accent width-100 mt-1">Add to Cart</button></a>
+            </div>  
+        </div>
+        <div class="card box-shadow" id="card-block">
+            <i class="fa fa-times dismiss"></i>
```
Comment (pankaj-prog) [inline Wishlist/wishlist.html]: id should not be repeated

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: Wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <img src="/assest/adidas.jpg" class="card-img" alt="card-image"/>
+            <div class="card-body">
+                <div class="card-title">American Tourister</div>
+                <div class="card-description">This is a new quality bag.</div>
+                <div class="price">Rs. 1234</div>
+                <a href="/Cart/cart.html"><button class="btn btn-accent width-100 mt-1">Add to Cart</button></a>
+            </div>  
+        </div>
+        <div class="card box-shadow" id="card-block">
```
Comment (SinchanaMS) [inline Wishlist/wishlist.html]: The same id has been used twice, they are supposed to be unique to one element. Please check.

File: Wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <li>
+                <a href="/Cart/cart.html"><div class="badge-wrapper">
+                <i class="bi bi-cart fa-lg"></i>
+                <span class="icon-badge flex-align-center">2</span>
+                </div></a>
+            </li>
+            <li><a href="/Wishlist/wishlist.html"><i class="bi bi-heart"></i></a></li>
+            <li><a href="#"><i class="bi bi-person-fill"></i></a></li>
+        </ul>
+    </div>
```
Comment (SinchanaMS) [inline Wishlist/wishlist.html]: Make use of nav instead of div for the navigation bar.

**Review by SinchanaMS — APPROVED** [review]
Suggested a couple of changes in the HTML file.
Otherwise, the UI looks really neat!

File: Wishlist/wishlist.html (hunk 3 above)
Comment (aiman-ansari) [reply Wishlist/wishlist.html]: Changed it

**Review by sanu29 — APPROVED** [review]
Good to merge

**Pull request merged**